test/unit/user_event/type_test.dart

```diff
@@ -197,12 +199,32 @@ void _typeTestHelper({bool hasDelay = false, bool isTextArea = false}) {
       });
     });
 
+  });
+
+  group('', () {
     testEventHandlerErrors(
       ['onInput'],
       (el) => hasDelay ? UserEvent.typeWithDelay(el, 'K', Duration(hours: 500)) : UserEvent.type(el, 'K'),
       isTextArea ? react.textarea : react.input,
     );
-  });
+
+    test('multiple event handler errors on every letter', () {
```

Comment:
I think a better description would be "event handlers that occur on delayed keystrokes" or something, since that's really what it's testing